### PR TITLE
Added to Release_notes.rst updates for July, August and September

### PR DESCRIPTION
**Pull request by vgladkov**

### CODE REVIEW

File: source/Release_notes.rst [hunk 1]
```diff
@@ -6,6 +6,43 @@ Release notes
...
+    study("My Script")
+    a = array.new_float(0)
+    for i = 0 to 5
+        array.push(a, close[i] - open[i])
+    plot(array.get(a, 4))
+
+August 2020
+--------------------------
+
+* Optimized Pine Scripts compilation time. Now scripts compiled from 1.5 to 2 times as fast as they did before the optimization.
```
Comment (LucFF) [inline source/Release_notes.rst]: Optimized script compilation time. Scripts now compile 1.5 to 2 times faster.

File: source/Release_notes.rst [hunk 2]
```diff
@@ -6,6 +6,43 @@ Release notes
 
 This page contains release notes of notable changes in Pine Script v4.
 
+September 2020
+--------------------------
+
+The following functions can now take a series as a length parameter:
```
Comment (LucFF) [inline source/Release_notes.rst]: The following functions now accept a series length parameter:

File: source/Release_notes.rst [hunk 3]
```diff
@@ -6,6 +6,43 @@ Release notes
...
+* `linreg <https://www.tradingview.com/pine-script-reference/v4/#fun_linreg>`__
+* `lowest <https://www.tradingview.com/pine-script-reference/v4/#fun_lowest>`__
+* `lowestbars <https://www.tradingview.com/pine-script-reference/v4/#fun_lowestbars>`__
+* `mom <https://www.tradingview.com/pine-script-reference/v4/#fun_mom>`__
+* `sma <https://www.tradingview.com/pine-script-reference/v4/#fun_sma>`__
+* `sum <https://www.tradingview.com/pine-script-reference/v4/#fun_sum>`__
+* `vwma <https://www.tradingview.com/pine-script-reference/v4/#fun_vwma>`__
+* `wma <https://www.tradingview.com/pine-script-reference/v4/#fun_wma>`__
+
+Added support for arrays and functions for working with them. Now you can use a powerful new array feature to build custom datasets. Read more on the `array manual <https://www.tradingview.com/pine-script-docs/en/v4/essential/Arrays.html>`__ page::
```
Comment (LucFF) [inline source/Release_notes.rst]: Added support for arrays and functions for working with them. You can now use the powerful new array feature to build custom datasets. See our `User Manual page on arrays <https://www.tradingview.com/pine-script-docs/en/v4/essential/Arrays.html>`__::

File: source/Release_notes.rst (hunk 2 above)
Comment (vgladkov) [reply source/Release_notes.rst]: Add link to our blog. Take a look, please.

**Review by LucFF — APPROVED** [review]

**Review by LucFF — APPROVED** [review]

**Pull request merged**